src/command_modules/azure-cli-batchai/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.4.3
++++++
+* Minor fixes
```

Comment:
This should call out that it fixes the bug since that's a customer-facing change (anyone who's run into that will want to know it's fixed).

Reply:
resolved